src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -350,7 +351,10 @@ export class SavedObjectsRepository {
             error: {
               id,
               type,
-              error: SavedObjectsErrorHelpers.createConflictError(type, id).output.payload,
+              error: {
+                ...SavedObjectsErrorHelpers.createConflictError(type, id).output.payload,
+                metadata: { isNotOverwritable: true },
```

Comment:
Well, the `output.payload` is generated by [Boom](https://hapi.dev/module/boom/api/?v=7.2.0), so if we went that route we'd have to expose a custom method to generate the types of payloads we want. Since we only currently want to have error metadata for this single purpose, I thought it was better to attach the metadata this way as it required the fewest amount of changes. I'd be happy to change it if the platform team wants to, though.

Reply:
Current implem is alright imho